### DISCUSSION TITLE
The docker container got hacked on my server

### DISCUSSION THREAD

**Discussion by Casraw** [AbdBarho/stable-diffusion-webui-docker#688] [General]
The docker image was quickly hacked. And something "simplefix" runs on it is for bitcoin mining. So it was written in the README of the downloaded files into the container. It was all in the /tmp folder. Only a short time it was running then I stopped the container.... Didnt delete the container because maybe someone wants to look into it...

**Comment by DevilaN** [comment]
Did you made public accessible instance of it on Internet?

**Comment by AbdBarho** [comment]
Yeah this container is not by any means intended for production usage, and if you absolutely have to, use some proxy container in between to add auth and make sure not everyone has access to the entire container. Something like https://oauth2-proxy.github.io/oauth2-proxy/
